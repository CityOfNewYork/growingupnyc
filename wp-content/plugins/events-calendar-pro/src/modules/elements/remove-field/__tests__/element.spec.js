import React from 'react';
import renderer from 'react-test-renderer';
import RemoveField from '../element';

describe( 'RemoveField', () => {
	test( 'should render component', () => {
		const component = renderer.create(
			<RemoveField onClick={ jest.fn() } />
		);
		expect( component.toJSON() ).toMatchSnapshot();
	} );
} );
