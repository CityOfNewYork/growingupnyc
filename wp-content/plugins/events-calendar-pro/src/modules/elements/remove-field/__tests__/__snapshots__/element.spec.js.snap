// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RemoveField should render component 1`] = `
<button
  className="tribe-editor__events-pro__remove-field"
  onClick={[MockFunction]}
  type="button"
>
  <<svg>icon mock</svg> />
</button>
`;
